packages/api-headless-cms/src/crud/contentModelGroup/beforeDelete.ts

```diff
@@ -5,14 +5,14 @@ import { CmsGroupPlugin } from "~/plugins/CmsGroupPlugin";
 import WebinyError from "@webiny/error";
 
 interface AssignBeforeGroupDeleteParams {
-    onBeforeDelete: Topic<BeforeGroupDeleteTopicParams>;
+    onGroupBeforeDelete: Topic<BeforeGroupDeleteTopicParams>;
```

Comment:
This is much better. But the conjunction is still not inline with the majority of other operations

More explicit never hurt anyone, I'm tempted to suggest `onBeforeContentModelGroupDelete`